lib/sycamore/sycamore/tests/unit/test_writer.py

```diff
@@ -120,10 +119,9 @@ def noop_map(doc: Document) -> Document:
 class TestDocSetWriter:
     def test_opensearch(self, mocker):
         context = mocker.Mock(spec=Context)
-        context.config = Config()
         docset = DocSet(context, mocker.Mock(spec=Node))
         execute = mocker.patch.object(OpenSearchWriter, "execute")
-        docset.write.opensearch(os_client_args={}, index_name="index")
+        docset.write.opensearch(os_client_args={}, index_name="index", index_settings={})
```

Comment:
why did you need to add index_settings here?  I feel like both that and os_client_args should default to empty dictionaries if unspecified.

Reply:
This is a Mock artifact, without this it tries to read index_settings from the mocked context which it turn returns a mock. Since we aren't really testing the writer logic here, this was a shorthand